logs/parse.py

```
@@ -0,0 +1,1441 @@
+
+    if (lpth == 6)  and (path[0] in ["CloudTrailEvent"]) and (path[1] in ["userIdentity"]) and (path[2] in ["sessionContext"]) and (path[3] in ["sessionIssuer"]) and (path[4] in ["userName"]) :
+        if path[5] in ["aws:ec2-infrastructure",
+                        "aws:ec2-instance",
+                        "AWSServiceRoleForAmazonSSM"
+                        "AWSServiceRoleForAutoScaling",
+                        "AWSServiceRoleForCloudWatchApplicationSignals",
+                        "AWSServiceRoleForConfig",
+                        "AWSServiceRoleForResourceExplorer",
+                        "github",
+                        "ssm-swarms-role"]:
+            interesting =1
```

Ruff (0.14.8)

Local variable `interesting` is assigned to but never used

Remove assignment to unused variable `interesting`

(F841)